fuzz/run.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+handle_fail()
+{
+  let "fails++"; mkdir "$fails" ; cat out ; cat out >> fails ; mv out "$fails" ;
```

Comment:
`cat out >> fails ;`
Please add some separator of individual runs. If I'm not mistaken, this will concatenate all logs and error reports from all runs into single file one after each other without any separator line e.g. `\n\n`

Reply:
at least for me cat supplies the newline, however there is still missing number of the failed instance, so I will add something